flink-ml-benchmark/src/test/java/org/apache/flink/ml/benchmark/BenchmarkTest.java

```diff
@@ -54,6 +60,44 @@ public void testParseJsonFile() throws Exception {
         assertTrue(benchmarks.containsKey("KMeansModel-1"));
     }
 
+    @Test
+    public void testJsonFileLegality() throws IOException, ClassNotFoundException {
+        File resourcesDir =
+                new File(
+                                this.getClass()
+                                        .getClassLoader()
+                                        .getResource("benchmark-demo.json")
+                                        .getPath())
+                        .getParentFile();
+        File[] jsonFiles =
+                resourcesDir.listFiles(
+                        (dir, name) ->
+                                name.endsWith(".json") && !name.equals("benchmark-demo.json"));
```

Comment:
It might be a bit simpler to remove `name.equals("benchmark-demo.json")` since the overhead of parsing this file is negligible compared to all the other work we need to do in the CI.

Reply:
The `benchmark-demo.json` is not included because some cases in the file like `Unmatch-Input` has invalid paramMap.  It is by design which makes it cannot be parsed correctly.

Reply:
I see. Sounds good.